tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,39 +6,26 @@
    "source": [
```

Comment:
<p>I like how you explain these concepts but I think it's not necessary as part of the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>